deps/deps-windows.cmake

```diff
@@ -27,16 +27,12 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL Clang)
     set(DEP_BOOST_TOOLSET "clang-win")
 endif ()
 
-if (${DEPS_BITS} EQUAL 32)
-    set(DEP_MSVC_GEN "Visual Studio ${DEP_VS_VER}")
```

Comment:
`DEP_MSVC_GEN` is still used in deps/CMakeLists.txt, will this break the build?

Reply:
Added back, thanks!